src/locales/default.json

```diff
@@ -160,6 +163,7 @@
     "ownerFeesTooltip": "Liquidity providers earn dynamic swap fees on every trade utilizing the Liquidity in this pool. Dynamic swap fees are controlled by the pool owner.",
     "payload": "Payload",
     "pendingClaims": "Pending claims",
+    "pendingCurrentWeek": "Pending: current week (estimate)",
```

Comment:
I suggest we use `Pending (estimate)` as the label, since the number will include both the current week and the previous one until the previous week results have been made official. So:
```
    "pendingCurrentWeek": "Pending (estimate)",
```